BuildScript.Nuke/GenerateSbom/SolutionSbomGenerator.cs

```diff
@@ -116,25 +118,55 @@ public void Generate (Tool cycloneDxTool, IReadOnlyCollection<ProjectInfo> proje
 
       _outputFile.Parent.CreateDirectory();
 
-      File.Copy(_workingDirectory / cleanedSbomFileName, _outputFile, overwrite: true);
+      //The sbom cleaner only works with xml files, so we need to convert the sbom if we want a different format.
+      //Given a requested xml file, we can just copy
+      var outputSbomFileToCopy = _workingDirectory / cleanedSbomFileName;
```

Comment:
We use `_workingDirectory / cleanedSbomFileName` two times to invoke the cleaner. As those expressions must match this one, I would move this variable declaration up above the Cleaner invocation and replace the other usages